src/test/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTaskTest.java

```diff
@@ -978,4 +978,19 @@ private static long getNumberFromLine(String line) {
         return Long.parseLong(matcher.group(1));
     }
 
+    @Test
+    public void testMaxWarningDefault() throws IOException {
+        final CheckstyleAntTask antTask = getCheckstyleAntTask();
+        final File inputFile = new File(getPath(WARNING_INPUT));
+        final Location fileLocation = new Location("build.xml", 42, 10);
+
+        antTask.setFile(inputFile);
+        antTask.setLocation(fileLocation);
+        antTask.execute();
+
+        assertWithMessage("File location should be valid")
+                .that(antTask.getLocation())
+                .isNotNull();
```

Comment:
ok , it works, but looks very unclear.

how check for location
helps with 
```
    <mutator>org.pitest.mutationtest.engine.gregor.mutators.experimental.MemberVariableMutator</mutator>
    <description>Removed assignment to member variable maxWarnings</description>
    <lineContent>private int maxWarnings = Integer.MAX_VALUE;</lineContent>
```

can we put in assert more meaningful checking ?